blocks/cards/cards.css

```diff
@@ -12,9 +12,12 @@
   background-color: var(--background-color);
 }
 
-.cards .cards-card-body {
+.cards .cards-card-body  {
   margin: 16px;
 }
+.anchor-text{
```

Comment:
Always keep the parent reference before we use these kind of class names. This can lead to overrides when we have the same class in generic